src/main/java/org/icgc/argo/program_service/controller/ProgramController.java

```diff
@@ -233,14 +233,14 @@ public void updateUser(
     }
   }
 
-  @GetMapping(value = "/joinProgramInvite")
+  @GetMapping(value = "/joinProgramInvite/{invite_id}")

```

Comment:
in swagger /programs/joinProgramInvite uses invite Id as a parameter. I am not very good at Java. Just want to make sure if there is a difference between endpoint using paramenter or end point that uses a "/" like shortname.
![Screenshot 2023-10-11 at 12 39 03 PM](https://github.com/icgc-argo/program-service/assets/79996555/d4609b33-77ec-4f42-8d3d-23e23225ce04)
![Screenshot 2023-10-11 at 12 38 53 PM](https://github.com/icgc-argo/program-service/assets/79996555/f769356b-9607-4cda-83da-b873ec79e7e9)

Reply:
I see the url is updated to this format. All good!: 
![Screenshot 2023-10-11 at 12 46 14 PM](https://github.com/icgc-argo/program-service/assets/79996555/e1fd334a-ea2d-4c09-a044-fce891e791dc)